torch/csrc/jit/fusers/cuda/simple_ops.h

```diff
@@ -0,0 +1,88 @@
+#if defined USE_CUDA && !(defined _WIN32) && !(defined USE_ROCM)
+#pragma once
+
+#include "torch/csrc/jit/fusers/cuda/cuda_fuser.h"
+#include <unordered_map>
+
+namespace torch { namespace jit {
+
+// What is a simple mappable operator?  It is:
+//    - Has an output with the same sizes as its input
+//    - Single output
+//    - Can handle non-contiguous input
+//    - Produces contiguous output
+// Some of these restrictions may be relaxable, but you should
+// carefully read the code first, as we rely on these assumptions.
+std::unordered_set<NodeKind> simple_mappable = {
```

Comment:
This is a non-inline non-static map in a header. That's not a good idea. This is really needed only in the fuser pass, so I don't understand why did this end up in a header

Reply:
It has everything to do with the logic of the file it was inside of. It tells you which nodes are eligible for fusion, and in fact it doesn't even account for all of them, which might have more complicated rules. I don't see any reason to lift it out.

Reply:
The only thing the pass cares about is `bool isSimpleMap(Node* node)` for each node that's passed. It doesn't care about what the function is just if it is a simple map or not.
However, if you like it better the other way I'm happy if we put it back in. Don't think this needs to be a sticking point.